### PR TITLE
Update ruby version to latest stable and fix db seeds

### PR DESCRIPTION
**Pull request by dmitrytrager**
- [x] Update ruby version to latest stable
- [x] Fix broken seeds

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
-
 # Setup user/dev for example site
+developer = FactoryGirl.build :developer
```
Comment (dmitrytrager) [inline db/seeds.rb]: Seeds are curently broken.
I found that we need to have default developer before creating users and that developer should be found by api key.

**Comment by t-harps** [comment]
Thanks Dmitry, just for reference: All pull requests are into the dev branch, we only ever merge dev directly into master. 

Also looks like the ruby version needs to be updated in .travis.yml to 2.3.1 as well, Travis is the continuous integration service we use.

We'll give you some more defined tasks/stuff to work on once everything is confirmed.

File: db/seeds.rb (hunk 1 above)
Comment (t-harps) [inline db/seeds.rb]: Ah yeh we haven't updated the seeds in a while. There is also another expected default developer, with the api key found under secrets['coposition_api_key'] so probably worth adding that to the seeds as well.

**Comment by dmitrytrager** [comment]
Ok, got it. I can reopen to dev branch if you want me to do it.

**Pull request closed without merging**